src/sentry/replays/endpoints/project_replay_summary.py

```diff
@@ -0,0 +1,178 @@
+        "content": e.response.content if e.response is not None else None,
+    }
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Replays"])
+class ProjectReplaySummaryEndpoint(ProjectEndpoint):
+    owner = ApiOwner.REPLAY
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+
+    def __init__(self, **options) -> None:
+        storage.initialize_client()
+        super().__init__(**options)
+        self.features = [
+            "organizations:session-replay",
+            "organizations:replay-ai-summaries",
+            "organizations:gen-ai-features",
+        ]
+
+    def make_seer_request(self, url: str, post_body: dict[str, Any]) -> Response:
+        """Make a POST request to a Seer endpoint. Raises HTTPError and logs non-200 status codes."""
+        data = json.dumps(post_body)
+
+        try:
+            response = requests.post(
+                url,
+                data=data,
+                headers={
+                    "content-type": "application/json;charset=utf-8",
+                    **sign_with_seer_secret(data.encode()),
+                },
+            )
+            response.raise_for_status()  # Raises HTTPError for 4xx and 5xx.
+
+        except requests.exceptions.HTTPError as e:
+            logger.exception(
+                "Seer returned error during replay breadcrumbs summary",
+                extra={"url": url, **_get_request_exc_extras(e)},
+            )
+            return self.respond(status=e.response.status_code if e.response is not None else 502)
+
+        except requests.exceptions.Timeout as e:
+            logger.exception(
+                "Seer timed out when starting a replay breadcrumbs summary",
+                extra={"url": url, **_get_request_exc_extras(e)},
+            )
+            return self.respond(status=504)
+
+        except requests.exceptions.RequestException as e:
+            logger.exception(
+                "Error requesting from Seer when starting a replay breadcrumbs summary",
+                extra={"url": url, **_get_request_exc_extras(e)},
+            )
+            return self.respond(status=502)
+
+        # Note any headers in the Seer response aren't returned.
+        return Response(data=response.json(), status=response.status_code)
+
+    def get(self, request: Request, project: Project, replay_id: str) -> Response:
+        """Poll for the status of a replay summary task in Seer."""
+        if not all(
+            features.has(feature, project.organization, actor=request.user)
+            for feature in self.features
+        ):
+            return self.respond(status=404)
+
+        # Request Seer for the state of the summary task.
+        return self.make_seer_request(
+            SEER_POLL_STATE_URL,
+            {
+                "replay_id": replay_id,
+            },
+        )
+
+    def post(self, request: Request, project: Project, replay_id: str) -> Response:
```

Comment:
Is there any instrumentation for this endpoint? I see that you added a trace for `get_summary_logs`, but we're doing a lot in this endpoint and it'd be good to keep an eye on it.